lib/features/menu/coffeeMenu/widget/coffee_card_widget.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class CoffeeCardWidget extends StatelessWidget {
+  const CoffeeCardWidget({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Theme.of(context).colorScheme.onBackground,
```

Comment:
If you need 1-2 parameters from a `Container`, it's better to use a single responsibility widget for that, you could replace this container with `ColoredBox`. It is made for performance reasons.

https://lazebny.io/flutter-best-practices/